index.js

```diff
@@ -57,6 +60,22 @@ client.on('message', (message) => {
     message.reply(message.author.displayAvatarURL())
     ev.addField('avatar', true)
   }
+
+  if (message.content === '!Announcement' && message.channel.id === '953362393071775814'){
```

Comment:
Might think about naming these id's, `const ANNOUNCEMENT_CHANNEL = 953362393071775814` (assuming it's the announcement channel) for clarity